test/Sentry.Tests/SentrySdkTests.cs

```diff
@@ -447,10 +446,17 @@ public void CaptureMessage_SdkInitialized_IncludesScope()
         [Fact]
         public void Implements_Client()
         {
-            var clientMembers = typeof(ISentryClient).GetMembers(BindingFlags.Public | BindingFlags.Instance);
-            var sentrySdk = typeof(SentrySdk).GetMembers(BindingFlags.Public | BindingFlags.Static);
+            var clientMembers = typeof(ISentryClient)
+                .GetMembers(BindingFlags.Public | BindingFlags.Instance)
+                .Select(m => m.ToString())
+                .ToArray();
+
+            var sentrySdkMembers = typeof(SentrySdk)
+                .GetMembers(BindingFlags.Public | BindingFlags.Static)
+                .Select(m => m.ToString())
+                .ToArray();
 
-            Assert.Empty(clientMembers.Select(m => m.ToString()).Except(sentrySdk.Select(m => m.ToString())));
+            sentrySdkMembers.Should().Contain(clientMembers);
```

Comment:
Why was this refactor needed? Did it break?

Reply:
This test failed during development because I forgot to add a method to `SentrySdk`. The error message was indecipherable, however, so I improved the test so that it shows exactly which methods are missing.